rmw_fastrtps_cpp/CMakeLists.txt

```diff
@@ -53,7 +54,7 @@ ament_target_dependencies(rmw_fastrtps_cpp
 
 configure_rmw_library(rmw_fastrtps_cpp)
 
-ament_export_libraries(rmw_fastrtps_cpp fastcdr fastrtps)
+ament_export_libraries(rmw_fastrtps_cpp fastcdr fastrtps ${FastRTPS_LIBRARIES})
```

Comment:
This line already list the target `fastcdr` and `fastrtps` to be exported. The target seem to not have the `IMPORTED_LOCATION` property set but only `LOCATION`. ament/ament_cmake#81 adds support for considering the `LOCATION` property in that case.